CHANGELOG.txt

```diff
@@ -1,3 +1,37 @@
+Airflow 1.10.14, 2020-12-10
+----------------------------
+
+Bug Fixes
+"""""""""
+
+- BugFix: Tasks with ``depends_on_past`` or ``task_concurrency`` are stuck (#12663)
+- Fix issue with empty Resources in executor_config (#12633)
+- Fix: Deprecated config ``force_log_out_after`` was not used (#12661)
+- Fix empty asctime field in JSON formatted logs (#10515)
+- [AIRFLOW-2809] Fix security issue regarding Flask SECRET_KEY (#3651)
+- [AIRFLOW-2884] Fix Flask SECRET_KEY security issue in www_rbac (#3729)
+- [AIRFLOW-2886] Generate random Flask SECRET_KEY in default config (#3738)
```

Comment:
```suggestion
- [AIRFLOW-2886] Generate random Flask SECRET_KEY in default config (#3738)
```
https://github.com/apache/airflow/pull/3738 overrides https://github.com/apache/airflow/pull/3729 and https://github.com/apache/airflow/pull/3651

Reply:
I know but we still have the all commits in Changelog